asm/custom-functions/src/lib.rs

```diff
@@ -90,6 +113,10 @@ fn dungeon_global_key_count(scene_index: u16) -> u16 {
     unsafe { (*dungeonflag_global(scene_index))[1] & 0xF }
 }
 
+fn storyflag_get_to_value(flag: u16) -> u16 {
```

Comment:
```suggestion
fn storyflag_get_value(flag: u16) -> u16 {
```
or only `storyflag_get`, idk